### PR TITLE
refactor: change the streamed amount to total amount in NFT SVG

### PR DESCRIPTION
**Pull request by andreivladbrg**
Closes https://github.com/sablier-labs/v2-core/issues/638

### CODE REVIEW

**Comment by PaulRBerg** [comment]
@andreivladbrg can you please share an example of an SVG generated with the new code in this PR?

File: shell/generate-svg.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Notes:
-# - There are four input arguments: progress, status, streamed amount, and duration
+# - There are four input arguments: progress, status, total, and duration
```
Comment (PaulRBerg) [inline shell/generate-svg.sh:4]: Fine to say "total amount" here

File: src/libraries/NFTSVG.sol [hunk 2]
```diff
@@ -58,9 +58,9 @@ library NFTSVG {
         (vars.statusWidth, vars.statusCard) =
             SVGElements.card({ cardType: SVGElements.CardType.STATUS, content: params.status });
 
-        // Generate the streamed card.
-        (vars.streamedWidth, vars.streamedCard) =
-            SVGElements.card({ cardType: SVGElements.CardType.STREAMED, content: params.streamed });
+        // Generate the total card.
```
Comment (PaulRBerg) [inline src/libraries/NFTSVG.sol:61]: "total amount"

**Review by PaulRBerg — APPROVED** [review]
Generally LGTM. Paging @razgraf so he can double-check that the content looks good to him, too.

**Comment by PaulRBerg** [comment]
Seeing the image now makes me thing that "Amount" might have been clearer ..

**Comment by razgraf** [comment]
> Seeing the image now makes me thing that "Amount" might have been clearer ..

Looks good.
Q1: Although "Amount" can refer to a couple of different amounts (total, streamed, withdrawn) I think it's ok to leave it at "Amount" - seems clearer to me
Q2: Any value in adding the streamed amount on the side?

**Comment by PaulRBerg** [comment]
> I think it's ok to leave it at "Amount" - seems clearer to me

Agree

> Should we change the label?

Yes. Sorry @andreivladbrg 🙈

> Any value in adding the streamed amount on the side?

No. Nobody will notice it (most folks in this space have ADHD).

Let's wait until a user requests the streamed amount.

**Comment by andreivladbrg** [comment]
Updated the card, now it looks like this:

image

File: shell/generate-svg.sh [hunk 3]
```diff
@@ -12,7 +12,7 @@ set -euo pipefail
 # Load the arguments while using default values
 arg_progress=${1:-4235}
 arg_status=${2:-"Streaming"}
-arg_streamed=${3:-"1.23M"}
+arg_amoung=${3:-"1.23M"}
```
Comment (PaulRBerg) [inline shell/generate-svg.sh:15]: Typo

**Review by PaulRBerg — APPROVED** [review]
✔️

**Pull request merged**